src/invidious/comments.cr

```diff
@@ -573,10 +573,11 @@ def content_to_comment_html(content)
 
     if run["navigationEndpoint"]?
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
+        base_url = URI.parse(url)
         url = URI.parse(url)
 
         if url.host == "youtu.be"
-          url = "/watch?v=#{url.request_target.lstrip('/')}"
+          url = "youtube.com/watch?v=#{url.request_target.lstrip('/')}"
```

Comment:
note that by adding `youtube.com` here youtube URLs will redirect to youtube rather than invidious.

Reply:
Indeed I should have done as with `displayed_url` before, I correct this immediately